src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs

```diff
@@ -40,7 +41,8 @@ public BindAttribute(params string[] include)
         /// </param>
         public BindAttribute([NotNull] Type predicateProviderType)
         {
-            if (!typeof(IPropertyBindingPredicateProvider).IsAssignableFrom(predicateProviderType))
+            if (!typeof(IPropertyBindingPredicateProvider).GetTypeInfo()
+                    .IsAssignableFrom(predicateProviderType.GetTypeInfo()))
```

Comment:
Isn't there a provided extension method for this? http://packagesearch.azurewebsites.net/?q=IsAssignableFrom